feature-pack/src/main/resources/modules/system/layers/base/javax/ejb/api/main/module.xml

```diff
@@ -25,11 +25,11 @@
 <module xmlns="urn:jboss:module:1.3" name="javax.ejb.api">
     <dependencies>
         <!-- This dep is for javax.naming -->
-        <module name="javax.api" export="true"/>
-        <module name="javax.transaction.api" export="true"/>
-        <module name="javax.xml.rpc.api" export="true"/>
```

Comment:
In general 'export' just causes problems, no other modules export API's that they depend on. These other modules will still be added to deployments as a dependency, but they can now be excluded if needed.